app/assets/javascripts/transaction.js

```diff
@@ -78,13 +78,24 @@ window.ST.transaction = window.ST.transaction || {};
       .flatMapLatest(function (data) { return Bacon.$.ajaxPost(formAction, data); })
       .flatMapLatest(toOpResult);
 
+    var analyticsEventSent = formSubmitWithData
+      .flatMapLatest(function() {
+        var timeout = Bacon.later(3000, "timeout");
+        var response = Bacon.fromCallback(function(callback) {
+          amplitude.getInstance().logEvent(analyticsEvent[0], analyticsEvent[1], callback);
```

Comment:
If amplitude is not loaded (i.e. api key is not set) this will lead to null pointer. Maybe a JS abstraction for analytics that only calls through if amplitude is defined?